02-loop.md

```diff
@@ -114,7 +114,7 @@ The improved version of `print_characters` uses a [for loop](reference.html#for-
 to repeat an operation---in this case, printing---once for each thing in a collection.
 The general form of a loop is:
 
-~~~ {.python}
+~~~
```

Comment:
As far as I remember we are using `{.python}` for code that learners could type and will actually work when following the lesson. Since this isn't the case I drop it.

If I missed something about how we use `{.python}` sorry about it. If the documentation isn't good enough I will improve it.